index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav>
+      <a href=""><img src="images/Logo Placeholder.svg" alt="" /></a>
+
+      <a href=""><img src="images/Icon - Menu.svg" alt="" /></a>
+    </nav>
+    <header class="header-section">
```

Comment:
- [ ] Kindly refactor your code, and make sure the `nav` resides within the `header`. 😄 

_Please note that the `header` should only contain the top part of the webpage, like navigation, logo, search bar, etc. So you also have to refactor the main part of your webpage. Consider using a `section` tag instead._